### DISCUSSION TITLE
[Help] How to limit dropdown search results only to enabled items?

### DISCUSSION THREAD

**Discussion by cheesyMan** [fomantic/Fomantic-UI#1682] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Help Wanted
In a search selection dropdown like this: 
https://fomantic-ui.com/modules/dropdown.html#search-selection

if some items in the menu are **disabled**, they will be anyway included in the user's search result (if matching).
Is there a way to limit the search result ONLY to enabled items?

Thanks
Alberto

**Comment by ko2in** [comment, marked as answer]
As far as I know, there's no option and callback to achieve this. But, when you start typing to search, the special class name `filtered` will be added to the text container. You can use this class to hide the disabled item from your menu.
```
.ui.search.dropdown .text.filtered + .menu .disabled.item {
    display: none;
}
```
Here's fiddle (https://jsfiddle.net/ko2in/ur2awmdt/8/). Try to type 'A' and you'll only see one active item.

**Comment by cheesyMan** [comment]
> As far as I know, there's no option and callback to achieve this. But, when you start typing to search, the special class name `filtered` will be added to the text container. You can use this class to hide the disabled item from your menu.
> 
> ```
> .ui.search.dropdown .text.filtered + .menu .disabled.item {
>     display: none;
> }
> ```
> 
> Here's fiddle (https://jsfiddle.net/ko2in/ur2awmdt/8/). Try to type 'A' and you'll only see one active item.

Perfect! Thanks
